.github/workflows/unit.yml

```diff
@@ -147,9 +147,9 @@ jobs:
           mkdir build && cd build
           cmake .. -G Ninja -DEVE_OPTIONS="${{ matrix.cfg.opts }}" -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchain/${{ matrix.cfg.comp }}.${{ matrix.cfg.arch }}.cmake
       - name: Compiling Unit Tests
-        run:  cd build && ninja unit.meta.exe unit.arch.exe  unit.core.bit_cast.exe -j 5
+        run:  cd build && ninja unit.meta.exe unit.arch.exe  unit.core.bit_cast.exe unit.api.regular.wide.exe unit.core.simd_cast.exe -j 2
```

Comment:
I don't like this. Why is this happening. We agreed to reduce the test cases.

Reply:
You mean why `-j 2`? Although the test on `simd.cast` before the change required a lot of memory (It seems locally about 60 GB), now it requires about 6 GB. Other tests need memory too. It seems `unit.api.regular.wide.exe` needs about 6-7 GB, and a test from `unit.arch.exe` also needs similar amount of memory.  I guess if we do not want to drop a lot of test cases, it's better to reduce build concurrency.